documentation/docs/scenarios/ui5-sap-cp/Readme.md

```diff
@@ -0,0 +1,63 @@
+# Docu stub
+
+# Purpose
+Here we should describe what this scenario does: It builds a SAPUI5 based application and deploys
+the build result into a NEO account. Build is performed via mta -> node -> grunt -> sap-best-practices plugin
+In fact it is `grunt clean, build, lint`.
+
+# The following tools are used
+ - mta (jar file which must be present on the build server; later: via docker)
+ - npm (needs to be installed and configured on the build server: later: via docker)
+ - grunt (will be materialized through npm)
+ - sap best practices (will be materialized through npm)
+
+# Prerequisites (in the central environment)
```

Comment:
I would reference to the requirements of the underlying steps, i.e. mtaBuild and neoDeploy + list/link generic prerequisites

- project "Piper" requires a Jenkins (2.x) with pipeline plugins to run
- project "Piper" needs to be [registered in Jenkins](https://github.com/SAP/jenkins-library/blob/master/README.md) as a Global Shared Library
- prerequisites for the [MTA build](mtaBuild.md)
- prerequisites for the [deployment](neoDeploy.md)